src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/IncidentOffenderParty.kt

```diff
@@ -5,16 +5,44 @@ import jakarta.persistence.Entity
 import jakarta.persistence.FetchType
 import jakarta.persistence.JoinColumn
 import jakarta.persistence.ManyToOne
+import org.hibernate.annotations.JoinColumnOrFormula
+import org.hibernate.annotations.JoinColumnsOrFormulas
+import org.hibernate.annotations.JoinFormula
 
 @Entity
 @DiscriminatorValue("offender")
 class IncidentOffenderParty(
   id: IncidentPartyId,
-  role: String,
```

Comment:
role and outcome moved into subclasses (well, outcome only relevant to offender party)